README.md

```diff
@@ -693,13 +694,13 @@ $ docker run -d -it -p 7200:8000 -p 7201:8001 -p 7202:8002 \
 
 When you complete these steps, you will have multiple containers; one on each VM, and all connected to each other on the 'ml-cluster-network' network. All the containers will be part of same cluster.
 
-## How to join TLS enabled cluster
+## How to join HTTPS(TLS) enabled cluster
 
 This example shows how to join a node to a TLS enabled cluster. There are two prerequistes for this configuration, first is TLS enabled bootstrap host App servers, second is the CA certificate or certificate chain of the host.
 
 Below example uses docker stack for MarkLogic cluster deployment. It will create a docker stack named mlstack with two services named bootstrap and node2.
 
-1. Create a bootstrap host using below compose file:
+1. Create a bootstrap host using the compose file:
```

Comment:
Create a bootstrap host using the following compose file:

Reply:
Changes added.